modules/functional-test/src/main/java/com/intuit/wasabi/tests/service/RapidExperimentationTest.java

```diff
@@ -0,0 +1,328 @@
+package com.intuit.wasabi.tests.service;
+
+import java.util.ArrayList;
+import java.util.List;
+import java.util.UUID;
+
+import org.apache.http.HttpStatus;
+import org.testng.Assert;
+import org.testng.annotations.AfterClass;
+import org.testng.annotations.BeforeClass;
+import org.testng.annotations.Test;
+
+import com.intuit.wasabi.tests.library.TestBase;
+import com.intuit.wasabi.tests.library.util.Constants;
+import com.intuit.wasabi.tests.model.Application;
+import com.intuit.wasabi.tests.model.Assignment;
+import com.intuit.wasabi.tests.model.Bucket;
+import com.intuit.wasabi.tests.model.Experiment;
+import com.intuit.wasabi.tests.model.Page;
+import com.intuit.wasabi.tests.model.User;
+import com.intuit.wasabi.tests.model.factory.AssignmentFactory;
+import com.intuit.wasabi.tests.model.factory.BucketFactory;
+import com.intuit.wasabi.tests.model.factory.ExperimentFactory;
+
+/*******************************************************************************
+ * Copyright 2017 Intuit
+ * <p>
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not use this file except in compliance with
+ * the License. You may obtain a copy of the License at
+ * <p>
+ * http://www.apache.org/licenses/LICENSE-2.0
+ * <p>
+ * Unless required by applicable law or agreed to in writing, software distributed under the License is distributed on
+ * an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the License for the
+ * specific language governing permissions and limitations under the License.
+ *******************************************************************************/
+public class RapidExperimentationTest extends TestBase {
+    /**
+     * This test class contains functional tests that check the rapid experimentation functionality
+     * 
+     * Created on 05/23/2017
+     */
+    private static final int RAPID_EXP_MAX_USERS = 5; // max cap on number of users
```

Comment:
@sudheerkethireddy I agree but using constants to specify these values involve modifying the code to change these values. Making it configurable is a nice to have where you just have to make changes in config files. Also, you could have multiple property files. For example, you could use a separate property file for rapid experiment test and let the global config file remain as is. We probably need to start doing it for existing tests too where they could each have their own config file. 

I would still suggest loading these from a property file as this is a new test that we are adding and it will lead the way for other tests. Should be simple to load these values from a property file: https://www.mkyong.com/java/java-properties-file-examples/ 

The autumn library: https://github.com/intuit/Autumn also abstracts property loading as we use in Wasabi code as well. For example, refer to the `configure()` method in: https://github.com/intuit/wasabi/blob/develop/modules/assignment/src/main/java/com/intuit/wasabi/assignment/AssignmentsModule.java. There is a usage around `getProperty()` from autumn.